app/css/views/themes-new.css

```diff
@@ -0,0 +1,60 @@
+.new-theme {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: stretch;
+  position: absolute;
```

Comment:
I know it sounds lame but I haven't been able to make `height: 100%` work. If you have any idea, please let me know.